tests/test_plugin_ods_rp2_input.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 macanudo527
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from rp2.rp2_decimal import ZERO, RP2Decimal
+
+from dali.configuration import Keyword
+from dali.in_transaction import InTransaction
+from dali.out_transaction import OutTransaction
+from dali.plugin.input.ods.rp2_input import InputPlugin
+
+
+class TestRP2InputOds:
+    def test_btc_transactions(self) -> None:
+        plugin = InputPlugin(
+            configuration_path='input/test_ods_rp2_input.ini',
+            input_file='input/test_ods_rp2_input.ods',
```

Comment:
Can you use a file with two (or more) assets (e.g. [this one](https://github.com/eprbell/rp2/blob/main/input/crypto_example.ods) or similar)?

Reply:
Ok I will add the ETH transactions to this